deployment/terraform/container_service.tf

```diff
@@ -380,6 +380,12 @@ data "template_file" "app_logstash" {
     image                            = local.app_logstash_image
     log_group_name                   = "log${local.short}AppLogstash"
     aws_region                       = var.aws_region
+    opensearch_endpoint              = "${aws_opensearch_domain.opensearch.endpoint}:443",
```

Comment:
Need to change the hardcoded port number to variable var.logstash_port

Reply:
Done